manager/rpc/license.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package license;
+
+message PollingRequest{
+	string vendor = 1; // Xilinx or Cadence
+	string feature = 2; // Vivado, Xcelium, etc
+	int32 quantity = 3; // Number of licenses
+	string user = 4; // Username
+	string hash = 5; // Job ID of previous request by the same client
+}
+
+message PollingResponse{
+	bool acquired = 1; // status of the acquired license
```

Comment:
I chose not to use the grpc status codes for this because returning a status code from the Polling() function under `common/license.go` will terminate the grpc connection. I want to maintain the grpc connection while notifying the client from the server that a license is not available yet. In the event that the client requests for an invalid vendor or feature, I terminate the grpc connection from the server side with a `InvalidArgument` code sent to the client:
```
        if !licenseCounter.validLicense(recv.Vendor, recv.Feature) {
            return grpcStatus.Errorf(grpcCodes.InvalidArgument, "\"%s %s\" is an unsupported vendor feature combination", recv.Vendor, recv.Feature)
        }
```